### PR TITLE
Refactor Cart::checkQuantities()

### PR DESCRIPTION
**Pull request by michaelKaefer**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Minor code style changes in Cart::checkQuantities()
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | -
| How to test?  | -

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9068)

### CODE REVIEW

File: classes/Cart.php [hunk 1]
```diff
@@ -3789,11 +3789,13 @@ public function getSummaryDetails($id_lang = null, $refresh = false)
...
      *
-     * @param bool $return_product Return the Product with not enough quantity instead
+     * @param bool $return_product Return the first found product with not enough quantity
      *
-     * @return bool|Product Indicates if there is enough in stock
+     * @return bool|array If all products are in stock: true; if not: either false or an array
+     *                    containing the first found product which is not in stock in the
+     *                    requested amount
      */
     public function checkQuantities($return_product = false)
```
Comment (PierreRambaud) [inline classes/Cart.php]: We take this opportunity to change this variable in camelCase?

**Comment by michaelKaefer** [comment]
@PierreRambaud Thank you! I made another commit. A totally different question: can I remove a label from a pull request? In another PR I answered but I was not able to remove the label "Waiting for author feedback"? Thanks again!

**Comment by PierreRambaud** [comment]
As I know, these labels are set by administrators or [Preston](https://github.com/PrestaShop/prestonbot). In case of `Waiting for author feedback` someone ask for a feedback, and will remove this label when he will read it.

**Comment by mickaelandrieu** [comment]
Preston is a super administrator, even me - his proud father - can't stop him labeling 🏷

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Thanks again @michaelKaefer!

**Pull request merged**